data/legend/dataflow-config.yaml

```
@@ -0,0 +1,12 @@
+paths:
+  metadata: "$_/metadata"
+
+  tier: "$_/generated/tier"
+  tier/raw: "/some/other/storage/raw_lh5"
+  tier_dsp: "$_/generated/tier/dsp"
+
+  par: "$_/generated/par"
+  par/dsp": "$_/generated/par/dsp"
+
+  log: "$_/generated/log",
```

YAML.jl throws an error when trying to read this, not sure if the `python` YAML package also does that..
```suggestion
  log: "$_/generated/log"
```